dev/tools/shared/images.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "service_name": "",
+    "context_dir": ""
+  },
+  {
+    "service_name": "chrome-sandbox",
+    "context_dir": "examples"
```

Comment:
If you add the context_dir and dockerfile like suggest above, then the logic for in the `for image_info in images_to_build:` will change, and each of the services would need to explicitly give the path to the Dockerfile.

```
"context_dir": "examples/chrome-sandbox",
"dockerfile": "examples/chrome-sandbox/Dockerfile"
```